src/cpp/core/include/core/json/Json.hpp

```diff
@@ -64,6 +67,50 @@ bool isType(const Value& value)
       return false;
 }
 
+inline std::string typeAsString(json_spirit::Value_type type)
+{
+        if (type == ObjectType)
```

Comment:
Ah, this is just a way for me to ensure all of the `type == ...` statements line up.